tests/test_sdist.py

```diff
@@ -17,7 +17,6 @@
     params=[
         "fixtures/twine-1.5.0.tar.gz",
         "fixtures/twine-1.6.5.tar.gz",
-        "fixtures/twine-1.5.0.zip",
```

Comment:
No particular objection from me, but I want to make sure I understand: are we OK to drop the `.zip`-style sdist here because we test the zip path elsewhere?

(Given that the remaining two fixtures are both tarballs, is there a reason we don't drop one of those instead?)

Reply:
As reported in the commit message, the test uses minimal zip archives generated on the fly to test the same code paths covered by testing with the zip archive in the repository. I haven't yet finished investigating why there are two tar.gz sdists in the test fixtures.

Reply:
Apologies, I missed that commit message. Makes sense, thanks!